go1.16/base/Dockerfile.tmpl

```diff
@@ -31,7 +31,7 @@ RUN ln -s /usr/bin/pip3 /usr/bin/pip
 
 ARG GOLANG_VERSION=1.16.5
 ARG GOLANG_DOWNLOAD_URL=https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
-ARG GOLANG_DOWNLOAD_SHA256=b12c23023b68de22f74c0524f10b753e7b08b1504cb7e417eccebdd3fae49061
```

Comment:
There is a bug in the script

![image](https://user-images.githubusercontent.com/2871786/123827975-fe028880-d8f8-11eb-927c-d71c351bd057.png)